### ISSUE TITLE
Allow separating click events (up OR down)

### ISSUE THREAD

**Issue by kerryj89** [pqrs-org/Karabiner-Elements#3641]
I wanted to use my middle click to output a `ctrl` + `left click down` and let go of both when I let go of my middle click.

The key code works fine and in the event viewer you only see a "up" event when middle-click is let go, but for mouse events it outputs a "click" (e.g. down AND up immediately).

**Comment by tpj2018** [comment]
Yes please, this would be great.  Also when you can use a key/button press as a toggle for down or up. Currently i use Dwell Click app for this specific function.

my left shift sends F20 on a single press. This will hold the left click down.
Next shift press will release it.
